packages/config/src/discovery/ProjectDiscovery.ts

```diff
@@ -646,8 +654,8 @@ export class ProjectDiscovery {
       )
       .map((contract) => contract.name)
     return safesWithThisMember.length === 0
-      ? []
-      : ['Member of ' + safesWithThisMember.join(', ') + '.']
+      ? undefined
+      : '* Member of ' + safesWithThisMember.join(', ') + '.'
```

Comment:
It's better to use the format string instead of the addition operator

Reply:
fixed